### PR TITLE
Small README updates

### PR DESCRIPTION
**Pull request by dallincrane**
Just left some comments in the README so you could adjust what you want and close this PR.

I'm wondering if everything within the `Instructions` headings would read cleaner as bullet points? Not sure, just a thought!

Hopefully the feedback helps. This project is definitely one of our best!

ps. Didn't have time to code the Black Diamond myself but had no issues with the code up until that point — except one small warning thrown that you'll see commented in Step 4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,14 +175,17 @@ export default connect( mapStateToProps )( App );
...
 
-Following action types comes the action creators. In Redux, actions are plain objects containing a type (describing what happened) and any data that might be necessary to the action. Our first action creator will be a function named `increment` that takes in a parameter of `amount`. The function then returns an object with two properties: `amount` - set equal to the `amount` parameter, and `type` set equal to the `INCREMENT` action type.  Create a `decrement` function that mimics `increment`, the only difference being that `type` should now be equal to `DECREMENT`. Export both of these functions.
+Following action types comes the action creators. In Redux, actions are plain objects containing a type (describing what happened) and any data that might be necessary to the action. Our first action creator will be a function named `increment` that takes in a parameter of `amount`.
+
+<!-- This sentence needs to be cleaned up a bit -->
```
Comment (r-walsh) [inline README.md]: updated somewhat in the upcoming commit. Not sure if enough

File: README.md [hunk 2]
```diff
@@ -253,6 +261,7 @@ You should now be able to interact with all of the increment and decrement butto
 
 <summary>`src/App.js`</summary>
 
+<!-- redo and undo are not yet defined here -->
```
Comment (r-walsh) [inline README.md]: Good catch, fixed.

File: README.md [hunk 3]
```diff
@@ -171,14 +175,17 @@ export default connect( mapStateToProps )( App );
 Now that our application is talking to Redux we need to set up Redux to actually do the things we want it to do. We'll start by creating action types, creating action creators, and implementing increment/decrement logic.
 
 **Instructions**
-
+<!-- This comma series is hard to read. Splitting into three sentences could clear things up. -->
```
Comment (r-walsh) [inline README.md]: Switching all of these to bullets per your comment. I tend to over-bullet, now I guess I'm overcompensating in the other direction. 🥇

**Comment by r-walsh** [comment]
Feedback greatly appreciated! Making changes now

**Comment by r-walsh** [comment]
Attempted to address in https://github.com/DevMountain/redux-counter/commit/56123d2e2e6730800a74eba1b5a39f2cae8bf263

File: README.md (hunk 1 above)
Comment (dallincrane) [inline README.md]: Reads a lot easier. I wouldn't know how to make it any better than what you got!

**Pull request closed without merging**